editor.planx.uk/src/@planx/components/types.ts

```diff
@@ -29,4 +29,5 @@ export enum TYPES {
   Send = 650,
   Calculate = 700,
   Confirmation = 725,
+  NextSteps = 730,
```

Comment:
This works :heavy_plus_sign: There's no obvious method here, but as long as it's a new unique number it'll be good to go! 

One new thing the original docs don't cover I think is that this list of Component types is now also maintained in planx-core and will need to be updated here as well: https://github.com/theopensystemslab/planx-core/blob/main/src/types/component.ts

Reply:
Thanks, I've added as a new PR to planx-core:
https://github.com/theopensystemslab/planx-core/pull/53